pkg/operator/starter.go

```diff
@@ -227,11 +227,12 @@ var RevisionConfigMaps = []revision.RevisionResource{
 var RevisionSecrets = []revision.RevisionResource{
 	// these need to removed, but if we remove them now, the cluster will die because we don't reload them yet
 	{Name: "etcd-client"},
-	// this is needed so that the cert syncer itself can request certs.  It uses localhost
-	{Name: "kube-apiserver-cert-syncer-client-cert-key"},
 	{Name: "kubelet-client"},
 	// etcd encryption
 	{Name: "encryption-config", Optional: true},
+
+	{Name: "localhost-recovery-serving-certkey"},
```

Comment:
why is this moved?

Reply:
I don't think certsyncer kubeconfig is wired to be live reloaded, nor recovery so it needs to be revisioned.

Reply:
write that

Reply:
done